Training25/Program.cs

```diff
@@ -4,10 +4,34 @@
 // ------------------------------------------------------------------
 // Program.cs
 // Program on main branch.
+// T8: Program to check if a pasword is strong.
 // ------------------------------------------------------------------------------------------------
+using System.Text.RegularExpressions;
+using static System.Console;
 namespace Training25;
+
 internal class Program {
-   static void Main (string[] args) {
-      Console.WriteLine ("Hello, World!");
+   static void Main () {
+      do {
+         Clear ();
+         Write ("Enter password: ");
+         var input = ReadLine ();
+         if (!string.IsNullOrEmpty (input)) {
+            if (input.Length >= 6 && !input.Any (char.IsWhiteSpace) && input.Any (char.IsLower)
+               && input.Any (char.IsUpper) && input.Any (char.IsNumber) &&
+               Regex.IsMatch (input, @"(?=.*[!@#$%^&*()-+])"))
+               WriteLine ("Password is strong");
+            else WriteLine ("""
+               Invalid password!
+               Password must have atleast 6 characters
+               Password must not include space characters
+               Password must have atleast one lower case character
+               Password must have atleast one upper case character
+               Password must have atleast one numeric character
+               Password must have atleast one special character
+               """);
+         } else WriteLine ("Please enter a non-empty string!");

```

Comment:
Have user-friendly messages.
```suggestion
         } else WriteLine ("Password cannot be empty.");
```